src/components/Rating.tsx

```
@@ -0,0 +1,56 @@
+  };
+
+  const handleMouseEnter = (rating: number) => {
+    setHoverRating(rating);
+  };
+
+  const handleMouseLeave = () => {
+    setHoverRating(0);
+  };
+
+  return (
+    <div className={`flex gap-8 ${className}`} onMouseLeave={handleMouseLeave}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)